### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by Mo7amedSMahdi**
- Create index.html.
- Create main.js
- Create a javascript object to have the books.
- display the books list from the javascript object.
- Save the javascript object to local storage and retrieve it.
- Add add new books functionality.
- Add remove book functionality.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "type": "git",
+    "url": "git+https://github.com/Mo7amedSMahdi/Awesome-books.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Mo7amedSMahdi/Awesome-books/issues"
+  },
+  "homepage": "https://github.com/Mo7amedSMahdi/Awesome-books#readme",
```
Comment (GzimAsani) [inline package.json:19]: - [x] Please add a descriptive readme file. Use this as a template [Template](https://github.com/microverseinc/readme-template)

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+function addNewBook() {
+  BooksObject.push({
+    id: BooksObject.length,
+    title: book.value,
+    author: author.value,
+  });
+  updateLocalStorage();
+  loadContent();
+}
+/* eslint-disable */
```
Comment (GzimAsani) [inline js/main.js:33]: - [x] Try not to disable linters, remember they are there to help you write a better code

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Mo7amedSMahdi  and @aboongm             
Good job so far! You are almost done :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the gitflow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check the comments below :arrow_double_down: _

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 2 above)
Comment (Mo7amedSMahdi) [reply js/main.js:33]: Dear GzimAsani,
in the project requirement, there is a tip for disabling the eslint errors for unused variable or using multiple classes and that is why I disable eslint for this line as this function is used in the remove button.

https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md

Comment (GzimAsani) [reply js/main.js:33]: Hello @Mo7amedSMahdi, thank you for your clarification. Sorry for the confusion

Comment (Mo7amedSMahdi) [reply js/main.js:33]: Deare Mr. @GzimAsani, Thanks for understanding, and sorry about forgetting the Readme file I just add it and I will submit it again.

Comment (GzimAsani) [reply js/main.js:33]: That would all. You've done great work :+1:

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Mo7amedSMahdi, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**